adapter/internal/discovery/xds/server.go

```diff
@@ -301,6 +303,7 @@ func cleanMapResources(apiIdentifier string, organizationID string, toBeDelEnvs
 	deleteBasepathForVHost(organizationID, apiIdentifier)
 	delete(orgIDOpenAPIEnvoyMap[organizationID], apiIdentifier)  //delete labels
 	delete(orgIDAPIMgwSwaggerMap[organizationID], apiIdentifier) //delete mgwSwagger
+	delete(orgIDAPIvHostsMap[organizationID], apiIdentifier)     //delete vhosts
```

Comment:
this is not intentional right? shouldn't it be like as follows;
```
delete(orgIDAPIvHostsMap[organizationID], apiUUID + constants.ProdEnvType) 
delete(orgIDAPIvHostsMap[organizationID], apiUUID + constants. SandEnvType) 
```

Reply:
This is not needed here now. came from the initial PR. will remove it